src/Console/ConsoleInputOption.php

```diff
@@ -266,6 +276,34 @@ public function validChoice($value): bool
         return true;
     }
 
+    /**
+     * Does this option have a prompt set?
+     *
+     * @return bool
+     */
+    public function hasPrompt(): bool
+    {
+        return $this->prompt !== null && $this->prompt !== '';
+    }
+
+    /**
+     * Collect input from the user
+     *
+     * @param \Cake\Console\ConsoleIo $io A ConsoleIo used to ask questions.
+     * @return null|string
+     */
+    public function promptForInput(ConsoleIo $io): ?string
+    {
+        if (!$this->prompt) {
+            return null;
```

Comment:
Shouldn't this be filtered out before calling? I'm not sure prompt should return null.

Reply:
If I don't have this return here, then there are type errors with `askChoice()` and `ask()`

Reply:
I guess I'm suggesting that an invalid prompt shouldn't return null but should be a configuration error. Are we calling promptForInput() on all options regardless?